data_steward/spec/pages/file_transfer_procedures.md

```diff
@@ -1,43 +1,98 @@
 title: File Transfer Procedures
 template: page
 
-## Data Steward Registration
+## DATA STEWARD REGISTRATION
+This section is designed to outline the procedure for requesting creation of a Google Cloud Storage Bucket and identifying and registering an Organization’s Data Steward. The Google Cloud Storage Bucket is the place within the DRC system to which EHR data files will be uploaded. A Data Steward is the personnel who will upload Electronic Health Record (EHR) data files to the Data and Research Center (DRC) on behalf of their Organization. 
 
-1. For each site you are a data steward for, enter your information in the [Data Steward Contact List](https://docs.google.com/spreadsheets/d/1Slh4teXKBwtD_ZrTFEjUTVH3Jk5ii-tcSJxocZnAqdo/edit?usp=sharing)
+The process applies to all program partner organizations that are responsible for sharing participant data (e.g., EHR) with the DRC. It details the documentation required to request creation of a Bucket and to register a Data Steward with the DRC.
 
-1. Request a `pmi-ops.org` account with data steward access by sending an e-mail to `sysadmin@pmi-ops.org` with the following:
- 
-     * __Subject__ New User Request: Data Steward
-     * Attestation of having completed your institutional security training
-     * Attestation of having entered your information in the [Data Steward Contact List](https://docs.google.com/spreadsheets/d/1Slh4teXKBwtD_ZrTFEjUTVH3Jk5ii-tcSJxocZnAqdo/edit?usp=sharing)
-     * Attached a signed, scanned copy of the [Rules of Behavior for Privileged Use](https://docs.google.com/document/d/1E6bRJ4l7AclEkaFS4Tg2zt9u3WyFMOpu4-omMjhlTRM/edit?usp=sharing). Note that the entire document must be attached, not just the signed page 
+---
+## KEYWORDS
+   * __SOP:__ Standard Operating Procedure
+   * __DRC:__ Data and Research Center
+   * __EHR:__ Electronic Health Records
+   * __Bucket:__ Location within Google Cloud Environment a sites data is stored
+   * __Data Steward:__ Individual responsible for uploading EHR data
+---
+## RESPONSIBILITY
+Each Organization tasked with uploading EHR data files or other Participant data to the DRC is required to request creation of a Bucket and register personnel who will fulfil the Data Steward role. The Data Steward is an individual at an Organization responsible for uploading EHR data to their Organization’s specified bucket within the DRC. An Organization may designate two (2) Data Stewards to provide backup coverage.  The responsibility of de-provisioning access is each Organization’s.  De-Provisioning or the unregistration of personnel designated as Data Steward or of an Organization itself is completed by sending an email to sysadmin@pmi-ops.org.  It is the organization's designated Data Steward(s) which is/are responsible for adherence to this procedure.
```

Comment:
The responsibility of de-provisioning access is each Organization’s  -> Each Organization is responsible for de-provisioning access.